src/pages/SearchHelpPage.vue

```diff
@@ -28,7 +28,7 @@
     <h2>Search by license</h2>
     <p>
       Put <em>license:</em> in front of your search term to see only results
-      from the specified source. For example,
+      from the specified license. For example,
```

Comment:
Thanks for spotting this. I think it would make more sense in this case `with the specified license`, instead of `from the specified license` which is a bit odd.